pkg/global/vars/vars.go

```diff
@@ -0,0 +1,72 @@
+package vars
+
+import (
+	"os"
+	"regexp"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/rest"
+
+	"github.com/k8snetworkplumbingwg/sriov-network-operator/pkg/global/consts"
+)
+
+var (
+	ClusterType string
+
+	// PlatformsMap contains supported platforms for virtual VF
+	PlatformsMap map[string]consts.PlatformTypes
+
+	DevMode bool
```

Comment:
can you add short description for each var ?

Reply:
done